packages/maskbook/src/_locales/en/messages.json

```diff
@@ -754,5 +754,44 @@
     "plugin_pooltogether_winning_odds": "Winning odds:",
     "plugin_pooltogether_in": "in",
     "plugin_pooltogether_manage": "Manage",
-    "plugin_pooltogether_token_not_found": "Token not found!"
+    "plugin_pooltogether_token_not_found": "Token not found!",
+    "plugin_good_ghosting_game_duration": "Game Duration",
+    "plugin_good_ghosting_current_round": "Current Round",
+    "plugin_good_ghosting_recurring_deposit": "Deposit Per Round",
+    "plugin_good_ghosting_round_length": "Round Length",
+    "plugin_good_ghosting_pool_apy": "Pool APY",
+    "plugin_good_ghosting_pool_earnings": "Pool Earnings",
+    "plugin_good_ghosting_extra_rewards": "Extra Rewards",
+    "plugin_good_ghosting_total_saved": "Total Saved",
+    "plugin_good_ghosting_game_launched": "Game Launched",
+    "plugin_good_ghosting_join_round": "Join Round",
+    "plugin_good_ghosting_join_deadline": "Join Deadline",
+    "plugin_good_ghosting_deposit": "Deposit {{index}}",
+    "plugin_good_ghosting_deposit_deadline": "Deposit Deadline {{index}}",
+    "plugin_good_ghosting_waiting_round": "Waiting Round",
+    "plugin_good_ghosting_waiting_round_end": "Waiting Period Ends",
+    "plugin_good_ghosting_withdraw": "Withdraw",
+    "plugin_good_ghosting_all_players_status_winning": "Winning",
+    "plugin_good_ghosting_all_players_status_waiting": "Waiting",
+    "plugin_good_ghosting_all_players_status_ghost": "Ghosts",
+    "plugin_good_ghosting_all_players_status_dropout": "Drop-outs",
+    "plugin_good_ghosting_status_winning": "Winning",
+    "plugin_good_ghosting_status_waiting": "Waiting",
+    "plugin_good_ghosting_status_ghost": "Ghost",
+    "plugin_good_ghosting_status_dropout": "Withdrawn",
+    "plugin_good_ghosting_status_unknown": "Unknown",
+    "plugin_good_ghosting_status": "Status",
+    "plugin_good_ghosting_deposits": "Deposits made",
+    "plugin_good_ghosting_total_deposited": "Total deposited",
+    "plugin_good_ghosting_address": "Address",
+    "plugin_good_ghosting_not_a_participant": "Looks like you're not a participant in this game",
+    "plugin_good_ghosting_next_event": "Coming up next",
+    "plugin_good_ghosting_game_end": "This game has ended",
```

Comment:
```suggestion
    "plugin_good_ghosting_game_end": "This game has ended.",
```

Reply:
Will keep this one as is since it's a title

---

![image](https://user-images.githubusercontent.com/23722804/127024172-a40904dc-5095-47c6-8a7a-91ba2c9de3c4.png)